mono/metadata/mono-security.c

```diff
@@ -77,6 +77,8 @@ GetTokenName (uid_t uid)
 {
 	gchar *uname = NULL;
 
+#ifdef HAVE_PWD_H
```

Comment:
The `#ifdef HAVE_GETPWUID_R` check on wraps a small part of the code in this function, so there is usage of `struct passwd` exposed outside of the `HAVE_GETPWUID_R` check that needs to be wrapped with `#ifdef HAVE_PWD_H`.